packages/react-core/src/components/NotificationDrawer/NotificationDrawerListItem.tsx

```diff
@@ -27,7 +27,7 @@ export const NotificationDrawerListItem: React.FunctionComponent<NotificationDra
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   onClick = (event: React.MouseEvent) => undefined as any,
   tabIndex = 0,
-  variant,
+  variant = 'default',
```

Comment:
Not sure if this is a breaking change? I would consider it a bug fix as items aren't designed to work without a variant set. Also this will now align with the alert's default variant. Curious what others think.

Reply:
This is also still a beta component so it gives us flexibility with the API.